packages/nimble-components/src/rich-text-editor/testing/rich-text-editor.pageobject.ts

```diff
@@ -156,10 +156,45 @@ export class RichTextEditorPageObject {
             .map(el => el.textContent || '');
     }
 
+    public isFooterHidden(): boolean {
+        const footerSection = this.getFooter()!;
+        return window.getComputedStyle(footerSection).display === 'none';
+    }
+
+    public getEditorTabIndex(): string {
+        return this.getTiptapEditor()?.getAttribute('tabindex') ?? '';
+    }
+
+    public async hideFooter(): Promise<void> {
+        this.richTextEditorElement.setAttribute('footer-hidden', '');
+        await waitForUpdatesAsync();
+    }
+
+    public async setDisabledState(): Promise<void> {
+        this.richTextEditorElement.setAttribute('disabled', '');
+        await waitForUpdatesAsync();
+    }
```

Comment:
Try to keep thee api aligned when wrapping public apis in helpers. so `setFooterHidden(footerHidden: boolean)` and `setDisabled(disabled: boolean)`. I'd also recommend on this making them configurable as suggested in the example signature

Reply:
I have updated these methods as configurable as suggested.